src/components/tab-group/tab-group.test.ts

```diff
@@ -0,0 +1,291 @@
+import { elementUpdated, expect, fixture, waitUntil } from '@open-wc/testing';
+import { html } from 'lit';
+import { clickOnElement } from '../../internal/test';
+import { queryByTestId } from '../../internal/test/data-testid-helpers';
+import type SlTabPanel from '../tab-panel/tab-panel';
+import type SlTab from '../tab/tab';
+import type SlTabGroup from './tab-group';
+import type { HTMLTemplateResult } from 'lit';
+
+interface ClientRectangles {
+  body?: DOMRect;
+  navigation?: DOMRect;
+}
+
+const wait = async (delayInMs = 0): Promise<void> => {
+  return new Promise(resolve => {
+    window.setTimeout(() => resolve(), delayInMs);
+  });
+};
```

Comment:
There's a helper in `@open-wc/testing` called `aTimeout`. I believe it does the same as this.